execution/inc/leatherman/execution/execution.hpp

```diff
@@ -307,6 +307,50 @@ namespace leatherman { namespace execution {
         lth_util::option_set<execution_options> const& options = { execution_options::trim_output, execution_options::merge_environment, execution_options::redirect_stderr_to_null });
 
     /**
+     * Executes the given program.
+     * @param file The name or path of the program to execute.
+     * @param arguments The arguments to pass to the program. On Windows they will be quoted as needed for spaces.
+     * @param input A string to place on stdin for the child process before reading output.
+     * @param environment The environment variables to pass to the child process.
+     * @param pid_callback The callback that is called with the pid of the child process.
+     * @param timeout The timeout, in seconds. Defaults to no timeout.
+     * @param options The execution options.  Defaults to trimming output, merging the environment, and redirecting stderr to null.
+     * @return Returns a result struct.
+     */
+    result execute(
```

Comment:
Why "ignoring all output"?
The first overload does not ignore stdout/stderr; it does exactly what the one before does, apart from calling the pid_callback. That's why I didn't add anything to its documentation, apart from the pid_callback arg entry.